tests/gtest/avifsvttest.cc

```diff
@@ -53,7 +82,9 @@ TEST_P(SvtAv1Test, EncodeDecodeStillImage) {
     // Lossless is not supported in SVT-AV1 2.3.0.
     EXPECT_GT(testutil::GetPsnr(*image, *decoder->image), 60.0);
     // The following should pass starting with SVT-AV1 version 3.0.0.
-    // EXPECT_TRUE(testutil::AreImagesEqual(*image, *decoded));
+    if (kSvtAv1SupportsLossless) {
+      EXPECT_TRUE(testutil::AreImagesEqual(*image, *decoded));
+    }
```

Comment:
Optional: I wonder if lines 82-87 should become an if-else statement:

```
    if (kSvtAv1SupportsLossless) {
      EXPECT_TRUE(testutil::AreImagesEqual(*image, *decoded));
    } else {
      EXPECT_GT(testutil::GetPsnr(*image, *decoder->image), 60.0);
    }
```

Similarly for lines 140-145 below.

Reply:
Done.